ouster-ros/package.xml

```diff
@@ -38,6 +39,7 @@
   <exec_depend>curl</exec_depend>
   <exec_depend>spdlog</exec_depend>
   <exec_depend>libtins-dev</exec_depend>
+  <exec_depend>point_cloud_transport_plugins</exec_depend>
```

Comment:
Remove from dependencies?

```suggestion
```

Reply:
I don't know exactly how ROS differentiates between `build_depend`, `depend`, and `exec_depend`, but I think it makes sense to keep it that way.